listen.js

```diff
@@ -1,12 +1,9 @@
+const app = require("./app.js");
+const [PORT = 9090] = process.env;
+
+app.listen(PORT, () => console.log("Listening on port 9090"));
+
 const express = require("express");
 const app = express();
 
 app.use(express.json());
```

Comment:
These lines of code shouldn't be in this file, they should already be in your `app.js` file so you could remove them here